### PR TITLE
8344899: Map RT-nnnn bug IDs to JDK-mmmmmmm in JavaFX sources

### PR DESCRIPTION
**Pull request by Maran23**
This PR changes all `RT-XXXX` references to `JDK-XXXXXXX`.
It also removes all `http://javafx-jira.kenai.com/browse/` occurrences.

As discussed, this will help a lot when looking up old issues, especially since not everybody know how to do a lookup with the `RT-XXXX` number in the JIRA.

Thanks to @kevinrushforth who provided the mapping!
I wrote a small Java program that replaces all the occurrences. This includes the following files:
- `.java, .css, .m, .h, .cc, .vert, .jsl, .c, .cpp`

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8344899](https://bugs.openjdk.org/browse/JDK-8344899): Map RT-nnnn bug IDs to JDK-mmmmmmm in JavaFX sources (**Bug** - P5)

### Reviewers
 * [Kevin Rushforth](https://openjdk.org/census#kcr) (@kevinrushforth - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jfx.git pull/1648/head:pull/1648` \
`$ git checkout pull/1648`

Update a local copy of the PR: \
`$ git checkout pull/1648` \
`$ git pull https://git.openjdk.org/jfx.git pull/1648/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 1648`

View PR using the GUI difftool: \
`$ git pr show -t 1648`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jfx/pull/1648.diff">https://git.openjdk.org/jfx/pull/1648.diff</a>

</details>
<details><summary>Using Webrev</summary>

[Link to Webrev Comment](https://git.openjdk.org/jfx/pull/1648#issuecomment-2495695739)
</details>

### CODE REVIEW

File: modules/javafx.graphics/src/main/java/javafx/scene/paint/RadialGradient.java [hunk 1]
```diff
@@ -460,7 +460,7 @@ public Paint interpolate(Paint endValue, double t) {
      */
     @Override public int hashCode() {
         // We should be able to just call focusAngle.hashCode(),
-        // see http://javafx-jira.kenai.com/browse/JFXC-4247
+        // see JFXC-4247
```
Comment (Maran23) [inline modules/javafx.graphics/src/main/java/javafx/scene/paint/RadialGradient.java:463]: I'm not sure how to map this issue

File: apps/samples/Modena/src/main/java/modena/Modena.java [hunk 2]
```diff
@@ -174,7 +174,7 @@ public void restart() {
         // scene.getStylesheets().add(testAppCssUrl);
         stage.setScene(scene);
         stage.setTitle("Modena");
-//        stage.setIconified(test); // TODO: Blocked by http://javafx-jira.kenai.com/browse/JMY-203
+//        stage.setIconified(test); // TODO: Blocked by JMY-203
```
Comment (Maran23) [inline apps/samples/Modena/src/main/java/modena/Modena.java:177]: I'm not sure how to map this issue

File: apps/samples/Modena/src/main/java/modena/Modena.java [hunk 3]
```diff
@@ -292,7 +292,7 @@ private void rebuildUI(boolean modena, boolean retina, int selectedTab, final Sa
             tab5.setContent(new ScrollPane(combinationsTest =
                     (Node)FXMLLoader.load(Modena.class.getResource("CombinationTest.fxml"))));
 
-            // Customer example from bug report http://javafx-jira.kenai.com/browse/DTL-5561
+            // Customer example from bug report DTL-5561
```
Comment (Maran23) [inline apps/samples/Modena/src/main/java/modena/Modena.java:295]: I'm not sure how to map this issue

File: apps/samples/Modena/src/main/resources/modena/ScottSelvia.fxml [hunk 4]
```diff
@@ -39,7 +39,7 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 <?import javafx.scene.paint.*?>
 <?import javafx.scene.text.*?>
 
-<!-- Customer test case from bug http://javafx-jira.kenai.com/browse/DTL-5561 -->
+<!-- Customer test case from bug DTL-5561 -->
```
Comment (Maran23) [inline apps/samples/Modena/src/main/resources/modena/ScottSelvia.fxml:42]: I'm not sure how to map this issue

**Comment by kevinrushforth** [comment]
I see you ran into a few non-RT bugs in the old Kenai JIRA instance. All of those should have been scrubbed a long time ago. There is no meaningful mapping for any of them. I no longer even have the ability to look in the old database (unless it was archived 9+ years ago when it was decommissioned).

* `JFXC-` was used for the old JavaFXScript compiler (back in the JavaFX 1.x time frame), and now completely irrelevant.
* `DTL-` was used for an old JavaFXScript-based design tool, the precursor of what eventually became Scene Builder
* `JMY-` was used for the JemmyFX test harness, which still exists in the [openjdk/jfx-tests](https://github.com/openjdk/jfx-tests) repo, but wouldn't have been used for any of the tests in the main jfx repo

File: modules/javafx.graphics/src/test/java/test/javafx/css/StylesheetTest.java [hunk 5]
```diff
@@ -465,7 +465,7 @@ public void testRT_23140() {
     public void testRT_30953_parse() {
 
         try {
-            // Make sure RT-30953.css can be parsed, serialized and deserialized with the current code,
+            // Make sure JDK-8123343.css can be parsed, serialized and deserialized with the current code,
```
Comment (kevinrushforth) [inline modules/javafx.graphics/src/test/java/test/javafx/css/StylesheetTest.java:468]: This refers to the name of a file, so should be reverted.

File: modules/javafx.graphics/src/test/java/test/javafx/css/StylesheetTest.java [hunk 6]
```diff
@@ -484,31 +484,31 @@ public void testRT_30953_parse() {
     }
 
     @Test public void testRT_30953_deserialize_from_v4() {
-        // RT-30953-v4.bss was generated with version 4
+        // JDK-8123343-v4.bss was generated with version 4
```
Comment (kevinrushforth) [inline modules/javafx.graphics/src/test/java/test/javafx/css/StylesheetTest.java:487]: Should be reverted.

File: modules/javafx.graphics/src/test/java/test/javafx/css/StylesheetTest.java [hunk 7]
```diff
@@ -484,31 +484,31 @@ public void testRT_30953_parse() {
...
         Stylesheet ss = deserialize("RT-30953-v4.bss");
         checkConvert(ss);
     }
 
     @Test
     public void testRT_30953_deserialize_from_2_2_45() {
 
-        // RT-30953-2.2.4bss was generated with javafx version 2.2.45 from 7u??
+        // JDK-8123343-2.2.4bss was generated with javafx version 2.2.45 from 7u??
```
Comment (kevinrushforth) [inline modules/javafx.graphics/src/test/java/test/javafx/css/StylesheetTest.java:495]: Should be reverted.

File: modules/javafx.graphics/src/test/java/test/javafx/css/StylesheetTest.java [hunk 8]
```diff
@@ -484,31 +484,31 @@ public void testRT_30953_parse() {
...
         Stylesheet ss = deserialize("RT-30953-2.2.45.bss");
         checkConvert(ss);
     }
 
     @Test
     public void testRT_30953_deserialize_from_2_2_4() {
 
-        // RT-30953-2.2.4bss was generated with javafx version 2.2.4 from 7u10
+        // JDK-8123343-2.2.4bss was generated with javafx version 2.2.4 from 7u10
```
Comment (kevinrushforth) [inline modules/javafx.graphics/src/test/java/test/javafx/css/StylesheetTest.java:503]: Should be reverted.

File: modules/javafx.graphics/src/test/java/test/javafx/css/StylesheetTest.java [hunk 9]
```diff
@@ -484,31 +484,31 @@ public void testRT_30953_parse() {
...
         Stylesheet ss = deserialize("RT-30953-2.2.4.bss");
         checkConvert(ss);
     }
 
     @Test
     public void testRT_30953_deserialize_from_2_2_21() {
 
-        // RT-30953-2.2.21.bss was generated with javafx version 2.2.21 from 7u21
+        // JDK-8123343-2.2.21.bss was generated with javafx version 2.2.21 from 7u21
```
Comment (kevinrushforth) [inline modules/javafx.graphics/src/test/java/test/javafx/css/StylesheetTest.java:511]: Should be reverted.

**Review by kevinrushforth — COMMENTED** [review]
Looks good except for a few changes in `StylesheetTest.java` where the `RT-nnnnn` refers to a filename and thus should be reverted.

**Comment by Maran23** [comment]
> Looks good except for a few changes in `StylesheetTest.java` where the `RT-nnnnn` refers to a filename and thus should be reverted.

Checked the whole project with: `JDK-[\d]*\.[\w]` and `JDK-[\d]*-`. No other occurrences found other than the one mentioned from you. Should be good now.

**Review by kevinrushforth — APPROVED** [review]

**Comment by Maran23** [comment]
Solved the merge conflict with `GlassView3D.m`.

**Review by kevinrushforth — APPROVED** [review]

**Comment by andy-goryachev-oracle** [comment]
Thank you @Maran23  so much for the cleanup!

**Pull request closed without merging**